packages/table/src/interactions/selectable.tsx

```diff
@@ -93,6 +101,21 @@ export class DragSelectable extends React.Component<IDragSelectableProps, {}> {
             return false;
         }
 
+        const regionCardinality = Regions.getRegionCardinality(region);
+        let focusCellCoordinates: IFocusedCellCoordinates = null;
+
+        if (regionCardinality === RegionCardinality.FULL_TABLE) {
+            focusCellCoordinates = { col: 0, row: 0 };
+        } else if (regionCardinality === RegionCardinality.FULL_COLUMNS) {
+            focusCellCoordinates = { col: region.cols[0], row: 0 };
+        } else if (regionCardinality === RegionCardinality.FULL_ROWS) {
+            focusCellCoordinates = { col: 0, row: region.rows[0] };
+        } else if (regionCardinality === RegionCardinality.CELLS) {
```

Comment:
i'd pull this out to a helper function and implement with a `switch` that simply `return`s in each case.